framework/test/mocha/common/utils/wait_for.js

```diff
@@ -51,6 +51,7 @@ function blockchainReady(retries, timeout, baseUrl, doNotLogRetries, cb) {
 
 	baseUrl = baseUrl || __testContext.baseUrl;
 
+	// eslint-disable-next-line wrap-iife
```

Comment:
I didn't want to rewrite the whole thing 😉 
And this will be removed while we move jest, so I kept it as it is for now

Reply:
Your call. But in this case, the issues's estimation is too high. Disabling rules is very bad practise instead of keeping them together in the `.eslint.json` file. It will be more complicated to track.

Reply:
at the same time, we won't add a new code like this, and I think that should be better =)
Definitely estimation was too heigh 😉